src/test/java/com/ericsson/ei/test/utils/TestConfigs.java

```diff
@@ -48,40 +49,25 @@ public static synchronized void init() throws Exception {
     private static synchronized void setUpMessageBus() throws Exception {
         LOGGER.debug("Debug:setting up message buss");
 
-        LOGGER.debug("before setting up message buss: amqpBroker: " + amqpBroker + ", conn: " + conn + ",cf:" + cf);
-        if (amqpBroker != null || conn != null || cf != null) {
+        LOGGER.debug("before setting up message buss: amqpBroker: " + amqpBroker + ", conn: " + connection + ",cf:"
+                + connectionFactory);
+        if (amqpBroker != null || connection != null || connectionFactory != null) {
             return;
         }
 
         int port = SocketUtils.findAvailableTcpPort();
-        System.setProperty("rabbitmq.port", "" + port);
-        System.setProperty("rabbitmq.user", "guest");
-        System.setProperty("rabbitmq.password", "guest");
-        System.setProperty("waitlist.initialDelayResend", "500");
-        System.setProperty("waitlist.fixedRateResend", "100");
+        setSystemProperties(port);
+        LOGGER.info("setting up message bus...");
```

Comment:
Remove this log message and change the level of the last log message to INFO. It could also be useful to log what port the message bus is using...

Reply:
Done... 
message bus port is now logged as info